libs/langchain-google-common/src/utils/anthropic.ts

```diff
@@ -0,0 +1,723 @@
+import {
+  ChatGeneration,
+  ChatGenerationChunk,
+  ChatResult,
+} from "@langchain/core/outputs";
+import {
+  BaseMessage,
+  BaseMessageChunk,
+  AIMessageChunk,
+  MessageContentComplex,
+  MessageContentText,
+  MessageContent,
+  MessageContentImageUrl,
+  AIMessageFields,
+  AIMessageChunkFields,
+} from "@langchain/core/messages";
+import {
+  ToolCall,
+  ToolCallChunk,
+  ToolMessage,
+} from "@langchain/core/messages/tool";
+import {
+  AnthropicAPIConfig,
+  AnthropicContent,
+  AnthropicContentText,
+  AnthropicContentToolUse,
+  AnthropicMessage,
+  AnthropicMessageContent,
+  AnthropicMessageContentImage,
+  AnthropicMessageContentText,
+  AnthropicMessageContentToolResult,
+  AnthropicMessageContentToolResultContent,
+  AnthropicRequest,
+  AnthropicRequestSettings,
+  AnthropicResponseData,
+  AnthropicResponseMessage,
+  AnthropicStreamContentBlockDeltaEvent,
+  AnthropicStreamContentBlockStartEvent,
+  AnthropicStreamInputJsonDelta,
+  AnthropicStreamMessageDeltaEvent,
+  AnthropicStreamMessageStartEvent,
+  AnthropicStreamTextDelta,
+  AnthropicTool,
+  AnthropicToolChoice,
+  GeminiTool,
+  GoogleAIAPI,
+  GoogleAIModelParams,
+  GoogleAIModelRequestParams,
+  GoogleAIToolType,
+  GoogleLLMResponse,
+} from "../types.js";
+
+export function getAnthropicAPI(config?: AnthropicAPIConfig): GoogleAIAPI {
+  // function notImplemented(): never {
```

Comment:
nit: remove

Reply:
Whoops. Yup. Removing. Thanks for that catch.